src/rt/sections_elf_shared.d

```diff
@@ -1043,15 +1043,13 @@ else
 
 version (LDC)
 {
-    version (PPC)
-    {
-        extern(C) void* __tls_get_addr_opt(tls_index* ti) nothrow @nogc;
-        alias __tls_get_addr = __tls_get_addr_opt;
-    }
-    else version (PPC64)
+
+version (linux) version (PPC) version = (linux_PPC_Any);
+version (linux) version (PPC64) version = (linux_PPC_Any);
+    
+    version (linux_PPC_Any)
     {
         extern(C) void* __tls_get_addr_opt(tls_index* ti) nothrow @nogc;
-        alias __tls_get_addr = __tls_get_addr_opt;
```

Comment:
This alias is still required. Also please indent the version declarations above (and remove the empty line before that block).